docs/shared/packages/next/plugin-overview.md

```diff
@@ -231,13 +231,53 @@ Next.js applications can be build with:
 nx build my-new-app
 ```
 
-And if you generated a library with --buildable, then you can build a library as well:
+And if you generated a library with `--bundler`, then you can build a library as well:
 
 ```shell
 nx build my-new-lib
 ```
 
-After running a build, the output will be in the `dist` folder. You can customize the output folder by setting `outputPath` in the project's `project.json` file.
+After running a build, the output will be in the `{workspaceRoot}/dist/{projectRoot}` folder.
+
+{% tabs %}
+
+{% tab label="Using inferred tasks" %}
+
+You can customize the output folder path by update the bundler's config. For example vite's config can be updated in `vite.config.ts`:
```

Comment:
I would also mention using `@nx/next/plugin` in `nx.json` and link to https://nx.dev/concepts/inferred-tasks#inferred-tasks-project-crystal page.